### DISCUSSION TITLE
User has an option to select alternative to email icon - envelope instead of @

### DISCUSSION THREAD

**Discussion by priyakdey** [nunocoracao/blowfish#183] [Ideas]
Hi, thanks for this beautiful theme; finally I could launch my personal website and add so much more features with much less effort in design.

Currently, the icon for email is `@` which to be honest, is the first time I have seen it. I have always seen the envelope icon being used.
Personally, I want to use the envelope icon, cause it is what I would want to see as a visitor (most ideas come from what I would prefer - I guess!).

Checking the structure, I can actually take a fork and change the SVG, but maybe like me many want an alternative.
So maybe we can have an option where the user can decide which icon to display for email.

**Comment by nunocoracao** [comment]
Hey @priyakdey thanks for the suggestion. That change is super easy to implement. Feel free to add the respective SVG icon and then a user can select that in the config file. Either open a PR or open an issue and I'll take care of it. Replacing the current icon doesn't seem like a good idea.

**Reply by priyakdey** [reply]
Sure thing, @nunocoracao  thank you for accepting this.
Yes, I was suggesting the same user might have an option for choosing which icon they want to use.
Sure will look into this and open a PR. 
Will open an issue against which I will raise a PR.

**Comment by nunocoracao** [comment]
@priyakdey changed this into an issue for implementation and created a branch for it if you want to contribute to it
